CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# [MongooseIM 4.0.0](https://github.com/esl/MongooseIM/releases/tag/4.0.0) - 2020-09-30
+## Overview
+MongooseIM 4.0 is all about putting a friendly face to MongooseIM’s amazing features.
+This includes the addition of an Erlang agnostic configuration allowing a broader pool of developers to benefit from MongooseIM.
+Logging and Kubernetes improvements to be more DevOps friendly and Manager friendly with load testing enabling managers to see the benefits of MongooseIM easily.
```

Comment:
DevOps-friendly, Manager-friendly... I even checked some pages about English and it seems that I'm right